style.css

```diff
@@ -0,0 +1,1194 @@
+html {
+	box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+	box-sizing: inherit;
+}
+	
+body {
+	min-height: 100vh;
+	background: #fff;
+	font-size: 12px;
+	line-height: 11px;
+	font-family: 'GothaPro';
+	color: #000;
+	margin: 0;
+}
+
+.container {
+	max-width: 1030px;
+	margin: 0 auto;
+}
+
+.site-navigation,
+.user-navigation,
+.catalog-list,
+.account-list,
+.customer-list,
+.miscellaneous-list,
+.main-footer__social {
+	list-style: none;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GerKateAxion/pullRequest?prid=5991795)